File: Codigos/Lista 08-09/3. Bela bonita.c
#include <stdio.h>

int main()
{
   int i;
   char a;
   float somatotal, v, D, P, C;
   
   for (i=1;i<=10; i++){
printf ("Qual a forma de pagamento? C - cartao; P - pix ; D - dinheiro: ");
fflush (stdin);
scanf("%s", &a);

switch(a){

case 'D':
 printf("Qual o valor %iº valor: ", i);
 fflush (stdin);
 scanf("%f", &v);
 somatotal+=v;
 D+=v;
break;
 case'P':
 printf("Qual o valor %iº valor: ", i);
 fflush (stdin);
 scanf("%f", &v);
 somatotal+=v;
 P+=v;
break;
case 'C':
 printf("Qual o valor %iº valor: ", i);
 fflush (stdin);
 scanf("%f", &v);
 somatotal+=v;
 C+=v;
break;

}
fflush (stdin);
}
printf("a soma de todas transacoes eh: %.1f \n",somatotal);
printf("Compras pagas no cartao: %.1f \n", C);
printf("Compras pagas no PIX: %.1f \n", P);
printf("Compras pagas em dinheiro em especie: %.1f \n", D);

    return 0;
}

File: Conversão para Python/Lista 08-09/3. Bela bonita.py
somatotal = 0
D = 0
P = 0
C = 0

for i in range(1, 11):
    a = input("Qual a forma de pagamento? C - cartão; P - pix; D - dinheiro: ")

    if a == 'D':
        v = float(input(f"Qual o valor do {i}º valor: "))
        somatotal += v
        D += v
    elif a == 'P':
        v = float(input(f"Qual o valor do {i}º valor: "))
        somatotal += v
        P += v
    elif a == 'C':
        v = float(input(f"Qual o valor do {i}º valor: "))
        somatotal += v
        C += v

print(f"A soma de todas transações é: {somatotal:.1f}")
print(f"Compras pagas no cartão: {C:.1f}")
print(f"Compras pagas no PIX: {P:.1f}")
print(f"Compras pagas em dinheiro em espécie: {D:.1f}")
